extensions/grpc/src/main/java/com/hazelcast/jet/grpc/GrpcServices.java

```diff
@@ -107,7 +108,7 @@ private GrpcServices() {
                 .withCreateContextFn(ctx -> channelFn.get().build())
                 .withCreateServiceFn((ctx, channel) -> new UnaryService<>(channel, callStubFn))
                 .withDestroyServiceFn(UnaryService::destroy)
-                .withDestroyContextFn(ManagedChannel::shutdown);
+                .withDestroyContextFn(channel -> channel.shutdown().awaitTermination(5, TimeUnit.SECONDS));
```

Comment:
what happens if times out 5 seconds?

Reply:
Ha! I thought it throws TimeoutException, but it just returns true/false if it managed to shutdown in time.

What should we do? Throw the exception ourselves?

Reply:
is there some way to terminate it not so gracefully?